libstorage/SQLBasicAccess.h

```diff
@@ -92,6 +92,10 @@ class SQLBasicAccess
 
 private:
     std::shared_ptr<SQLConnectionPool> m_connPool;
+    std::string const c_mediumBlobType = "mediumblob";

```

Comment:
this seems not used?

Reply:
There is something wrong when sync the latest code, already deleted the useless members.